azure/packages/azure-service-utils/api-report/azure-service-utils.beta.api.md

```diff
@@ -6,6 +6,4 @@
 
 export { IUser }
 
-export { ScopeType }
```

Comment:
Where did this change come from?

Reply:
It is called out - the last line of the description. ;)